standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1264,6 +1264,9 @@ public enum ConfVars {
         "hive.notification.sequence.lock.retry.sleep.interval", 10, TimeUnit.SECONDS,
         "Sleep interval between retries to acquire a notification lock as described part of property "
             + NOTIFICATION_SEQUENCE_LOCK_MAX_RETRIES.name()),
+    NOTIFICATION_ALTER_PARTITIONS_V2_ENABLED("metastore.alterPartitions.notification.v2.enabled",
+        "hive.metastore.alterPartitions.notification.v2.enabled", true,
```

Comment:
Should we disable this property by default? Whenever the consumers of these events are ready to process them, then they can enable it. We can probably enable this by default in Hive 5.0 release.

Reply:
As discussion offline, we can disable the property downstream, for upstream 4.0 release, we can enable it by default